src/features/defi/providers/foxy/components/FoxyManager/Withdraw/components/Confirm.tsx

```diff
@@ -97,14 +107,14 @@ export const Confirm: React.FC<StepComponentProps & { accountId?: AccountId | un
           type: state.withdraw.withdrawType,
           bip44Params,
         }),
-        foxyApi.getGasPrice(),
+        adapter.getGasFeeData(),
       ])
       dispatch({ type: FoxyWithdrawActionType.SET_TXID, payload: txid })
       onNext(DefiStep.Status)
 
       const transactionReceipt = await poll({
         fn: () => foxyApi.getTxReceipt({ txid }),
-        validate: (result: TransactionReceipt) => !isNil(result),
+        validate: (result: ethers.providers.TransactionReceipt) => !isNil(result),
```

Comment:
and here

Reply:
https://github.com/shapeshift/web/commit/6f6cb6777bd937c9a498160b7a1ad8d06af17e4a